sdk/src/oauth/state-machines/debug-oauth-2025-06-18.ts

```
@@ -781,7 +784,10 @@ export const createDebugOAuthStateMachine = (
               state.authorizationServerUrl,
             );
             let authServerMetadata = null;
-            let lastError = null;
+            // Every probe's outcome. Replaces a `lastError` the 4xx branch
+            // never wrote to, so an all-404 run (the ordinary case) threw
+            // `Last error: null`.
+            const metadataProbes: string[] = [];
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)